Activity.js

```diff
@@ -0,0 +1,19 @@
+class Activity {
+  constructor(category, description, minutes, seconds) {
+    this.category = category;
+    this.description = description;
+    this.minutes = minutes;
+    this.seconds = seconds;
```

Comment:
I think this is good. One thing I think we can keep in mind for the future is the possibility to change this.minutes and this.seconds to this.time and assign it to minutes plus seconds. We can address that if we need to in future iterations.